### PR TITLE
[bitnami/*] Add upgrade details to README

### LINKED ISSUE #4468: [bitnami/postgresql] Upgrade from 9.8.12 to 10.1.0 failed
**Which chart**:
The name (and version) of the affected chart
bitnami/postgresql 10.1.0

**Describe the bug**
The helm upgrade from 9.8.12 to 10.1.0 failed (using latest terraform helm provider). I think it would also happen on bare helm upgrade.

*error*
```
cannot patch "postgresql-postgresql" with kind StatefulSet: StatefulSet.apps "postgresql-postgresql" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
```

**To Reproduce**

1. deploy bitnami/postgresql 9.8.12 chart
2. try to upgrade to 10.1.0
3. see error 

**Expected behavior**
Should simply deploy the update

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3+k3s3", GitCommit:"0e4fbfefe1dd8734756dfa4f9ab4fc89665cece4", GitTreeState:"clean", BuildDate:"2020-11-13T07:19:02Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

- terraform versions
```hcl
terraform {
  required_providers {
    docker = {
      source  = "terraform-providers/docker"
      version = "2.7.2"
    }
    helm = {
      source  = "hashicorp/helm"
      version = "1.3.2"
    }
    kubernetes = {
      source  = "hashicorp/kubernetes"
      version = "1.13.3"
    }
  }
  required_version = ">= 0.13"
}
```

### PR DESCRIPTION
**Pull request by FraPazGal**
**Description of the change**

Added instructions for upgrading to the latest major reusing the PVC.

**Applicable issues**

  - fixes #4468

**Additional information**

**Checklist** 
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by carrodher** [comment]
Good catch! It seems it's something we also need to do in other Helm Charts with PostgreSQL as a dependency and probably in PostgreSQL HA as well, right?.

**Comment by FraPazGal** [comment]
Updated READMEs of those charts using bitnami/postgresql as dependency except bitnami/redmine which already has them.

I think this is not necessary for postgresql-ha, I haven't seen any changes done to it. Could you confirm this?

File: bitnami/discourse/README.md [hunk 1]
```diff
@@ -476,10 +476,54 @@ Find more information about how to deal with common errors related to Bitnami’
...
+```console
+$ export DISCOURSE_HOST=$(kubectl get svc --namespace default discourse --template "{{ range (index .status.loadBalancer.ingress 0) }}{{ . }}{{ end }}")
+$ export DISCOURSE_PASSWORD=$(kubectl get secret --namespace default discourse-discourse -o jsonpath="{.data.discourse-password}" | base64 --decode)
+$ export POSTGRESQL_PASSWORD=$(kubectl get secret --namespace default discourse-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+$ export POSTGRESQL_PVC=$(kubectl get pvc -l app.kubernetes.io/instance=discourse,app.kubernetes.io/name=postgresql,role=master -o jsonpath="{.items[0].metadata.name}")
+```
+
+##### Delete statefulsets
+
+Delete the Odoo deployment and delete the PostgreSQL statefulset. Notice the option `--cascade=false` in the latter:
```
Comment (carrodher) [inline bitnami/discourse/README.md:497]:
```suggestion
Delete the Discourse deployment and delete the PostgreSQL statefulset. Notice the option `--cascade=false` in the latter:
```

File: bitnami/postgresql/README.md [hunk 2]
```diff
@@ -525,7 +525,7 @@ $ helm upgrade my-release stable/postgresql \
 
 > Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
 
-### To 10.0.0
+## To 10.0.0
```
Comment (carrodher) [inline bitnami/postgresql/README.md:528]: This is a level inside `## Upgrading` so it should contain 3#, check others
```suggestion
### To 10.0.0
```

File: bitnami/postgresql/README.md [hunk 3]
```diff
@@ -552,6 +552,44 @@ $ helm upgrade my-release stable/postgresql \
 
 - The term `master` has been replaced with `primary` and `slave` with `readReplicas` throughout the chart. Role names have changed from `master` and `slave` to `primary` and `read`.
 
+To upgrade to `10.0.0`, it should be done reusing the PVCs used to hold the PostgreSQL data on your previous release. To do so, follow the instructions below (the following example assumes that the release name is `postgresql`):
```
Comment (carrodher) [inline bitnami/postgresql/README.md:555]: I can't select line 541 as it is out of the scope of this review, but this sentence doesn't have sense, can you remove it?

File: bitnami/postgresql/README.md (hunk 2 above)
Comment (FraPazGal) [reply bitnami/postgresql/README.md:528]: My bad, the error is on the "To 9.0.0" line.

**Review by carrodher — APPROVED** [review]

**Pull request merged**